resources/lang/en/app.php

```diff
@@ -110,4 +110,7 @@
   'alert.success.user_restored' => 'User restored successfully',
   'dashboard.reorder' => 'Reorder and pin items',
   'dashboard.settings' => 'Settings',
+  'role' => 'Authentification role',
+  'unauthorized_for_form' => 'You are not authorised to view this form.',
```

Comment:
You are not authorized to view this form.

Reply:
This spelling error is a question of American English or British English.

Should I use American English?

Reply:
fixed with [e6d8616](https://github.com/linuxserver/Heimdall/pull/1304/commits/e6d8616bf9315d50e80ce7dfb9135c0453485ba0)